ui/src/UserAdmin.js

```diff
@@ -350,6 +363,12 @@ class UserAdmin extends Component {
                 helperText={this.state.groupNameValidationError}
                 value={this.state.groupName}
               />
+              <TextField
+                fullWidth={true}
+                label="Description"
+                onChange={this.onGroupDescriptionChange}
+                value={this.state.groupDescription}
+              />
```

Comment:
The database column only allows for 255 characters, would be nice to enforce that in the front end to avoid a 500 error response.